models/runs.mdx

```diff
@@ -332,27 +332,74 @@ See [State fields](#run-states) for a full list of possible run states.
 
 ## View logged runs
 
-View a information about a specific run such as the state of the run, artifacts logged to the run, log files recorded during the run, and more. 
+View information about a specific run such as the state of the run, artifacts logged to the run, metrics, system metrics, and more.
 
-<Frame>
-    <img src="/images/runs/demo-project.gif" alt="Project navigation demo"  />
-</Frame>
+You can view and monitor runs using the W&B App or the `wandb beta leet` terminal UI.
 
-To view a specific run:
+<Tabs>
+<Tab title="App">
+
+To view a specific run in the W&B App:
 
 1. Navigate to the [W&B App](https://wandb.ai/home).
 2. Navigate to the W&B project you specified when you initialized the run.
 3. Within the project sidebar, select the **Workspace** tab.
 4. Within the run selector, click the run you want to view, or enter a partial run name to filter for matching runs.
 
-Note that the URL path of a specific run has the proceeding format:
+<Frame>
+    <img src="/images/runs/demo-project.gif" alt="Project navigation demo"  />
+</Frame>
+
+The URL path of a specific run has the proceeding format:
 
 ```text
 https://wandb.ai/<team-name>/<project-name>/runs/<run-id>
 ```
 
 Where values enclosed in angle brackets (`< >`) are placeholders for the actual values of the team name, project name, and run ID.
 
+</Tab>
+<Tab title="LEET">
+
+To view a run locally in your terminal using the `wandb beta leet` terminal UI:
+
+1. If you started the run locally from a script, navigate to the directory where you ran your code. It contains a `wandb/` directory with a subdirectory per run and a `latest-run/` symbolic link. Each run directory contains a transaction log with the suffix `.wandb`.
```

Comment:
Let's add that, in addition to the `.wandb` suffix, the run ID is prefixed to this file (`run-id.wandb`)